.eslintrc.js

```diff
@@ -28,6 +35,18 @@ module.exports = {
     // The rule is also mostly irrelevant to this codebase due to TypeScript
     // usage (where .tsx is required).
     "react/jsx-filename-extension": [0],
+    "@typescript-eslint/naming-convention": [
+      ...airbnbTypeScriptNamingConventionRules,
+      // Allow underscore-only identifiers to indicate ignored positional variables.
+      {
+        selector: "variable",
+        format: null,
+        custom: {
+          regex: "^_+$",
+          match: true,
+        },
+      },
```

Comment:
Looks like this has to be:

```suggestion
      {
        selector: "variable",
        format: null,
        custom: {
          regex: "^_+$",
          match: true,
        },
      },      {
        selector: "variable",
        format: [],
        filter: {
          regex: "^_+$",
          match: true,
        },
        custom: {
          regex: "^(_+)$",
          match: true,
        },
      },
```